cobra/io/sbml.py

```diff
@@ -697,7 +699,7 @@ def process_association(ass):
                     LOGGER.warning("Encoding OBJECTIVE_COEFFICIENT in "
                                    "KineticLaw is discouraged, "
                                    "use fbc:fluxObjective "
-                                   "instead: %s", cobra_reaction)
+                                   "instead: %s", reaction)
```

Comment:
This is necessary, because the loop is on reaction, so if it is not corrected, the output stays with the last reaction in a different loop.